packages/presentation/presentation_publishing/index.ts

```diff
@@ -39,6 +39,7 @@ export {
   type PublishesUnifiedSearch,
   type PublishesWritableUnifiedSearch,
 } from './interfaces/fetch/publishes_unified_search';
+export { apiPublishesReload, type PublishesReload } from './interfaces/fetch/publishes_reload';
```

Comment:
I don't think you needed to export this - it seems like you aren't using it anywhere

Reply:
Yep you are right. I am not using it. Since the way I do the manual reload below is fine, I will remove this.